snooty/parser.py

```diff
@@ -424,6 +438,60 @@ def dispatch_departure(self, node: docutils.nodes.Node) -> None:
                 target.children = [identifier]
                 item.term.append(target)
 
+    def handle_tabset(self, node: n.Directive, line: int) -> None:
```

Comment:
Do you need the `line` argument? You can always get a node's source line number with `node.start[0]`